assets/src/containers/App.js

```diff
@@ -12,7 +12,6 @@ function App (props) {
     return (window.location.href = user.loginURL)
   }
   // This is needed to support both the standalone and LTI tool
```

Comment:
Is this comment line still relevant? Is it related to the removed line or the 2 under?

Reply:
Comment here is not relevant anymore